.travis.yml

```diff
@@ -69,7 +69,7 @@ before_script:
     # Make sketchbook/libraries folder.
     - mkdir -p $CI_SRC_PATH/sketchbook/libraries
     - export CI_LIB_PATH=$CI_SRC_PATH/sketchbook/libraries
-    # Make rosserial_arduino in the sketchbook/libraries folder, including victoria_nav_msgs
+    # Make rosserial_arduino in the sketchbook/libraries folder, including victoria_nav_msgs,victoria_sensor_msgs
     - rosrun rosserial_arduino make_libraries.py $CI_LIB_PATH victoria_nav_msgs
```

Comment:
Ok. Removing them seems like the right choice.
The reason the comments aren't here is because when you push a new commit that changes that line, the comment disappears, unless you go that commit to see the old comment.
It's a way to see that something has been changed on that line during a review.
Like if you change the line, presumably the old comments are no longer valid.